src/components/home.js

```diff
@@ -507,37 +493,6 @@ class home extends Component {
     // console.log("Diable Audio: " + this.state.disableAudio);
     // console.log("Rotate Value: " + this.state.RotateValue);
 
-    const trims = this.state.trims.map((trim, i) => (
-      <Row gutter={10} key={i}>
-        <Col span={10}>
-          <Typography.Text strong style={{ paddingRight: "0.2rem" }}>
-            From
-          </Typography.Text>
-          <div className="form-group">
-            <Input
-              placeholder="hh:mm:ss"
-              id={`trim-${i}-from`}
-              value={trim.from}
-              onChange={this.onChange}
-            />
-          </div>
-        </Col>
-        <Col span={10}>
-          <Typography.Text strong style={{ paddingRight: "0.2rem" }}>
-            To
-          </Typography.Text>
-          <div className="form-group">
-            <Input
-              placeholder="hh:mm:ss"
-              id={`trim-${i}-to`}
-              value={trim.to}
-              onChange={this.onChange}
-            />
-          </div>
-        </Col>
-      </Row>
-    ));
```

Comment:
Hello Netx, Why did you remove this ? Did you test it locally after making all this changes ?

Reply:
It was an unused variable that was causing a warning on start.
![cmd_xJCOK3ke20](https://user-images.githubusercontent.com/30041127/70315917-78991e00-1812-11ea-8566-1271e4e5f099.png)